specs/change_password/01_main.rst

```diff
@@ -0,0 +1,165 @@
+=====  ===============  =========  ==========
+epic        title        author     created
+=====  ===============  =========  ==========
+`74`_  Change password  tschubotz  2019-01-17
+=====  ===============  =========  ==========
+
+.. _74: https://github.com/gnosis/safe/issues/74
+
+.. _Main:
+
+
+#. `Main`_
+#. `User Interface`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+---------------------
+
+* User wants to change the current app password.
+* Current password could be compromised and hence needs to be changed.
+
+Inputs
+-----------
+
+- Current password.
+- New password
+
+  - Minimum 8 characters (>=8)
+  - At least 1 number
+  - At least 1 letter
+  - No more than 2 identical characters in a row
+    (e.g. 111 is not allowed, 11 is fine)
+  - Maximum length?
```

Comment:
Do we need this?

Also any idea how to make sure that these requirements are always in sync with the ones mentioned in create password

Reply:
> Do we need this?

Your call. As long as some kind of overflow is prevented, we don't need it.

> Also any idea how to make sure that these requirements are always in sync with the ones mentioned in create password

Good point, will move this to a common file that is referenced :)